test/e2e/pages/create-move.js

```diff
@@ -152,11 +155,13 @@ class CreateMovePage extends Page {
    * @param {Object} personalDetails - personal details to fill form in with
    * @returns {Promise<FormDetails>} - filled in personal details
    */
-  async fillInPersonalDetails(personalDetails) {
-    await t.expect(this.getCurrentUrl()).contains('/move/new/personal-details')
+  async fillInPersonalDetails(personalDetails, { include, exclude } = {}) {
```

Comment:
Maybe document these new arguments and what they allow you to do

Reply:
Yep. Added.